config/ember-try.js

```diff
@@ -7,6 +7,105 @@ module.exports = {
       }
     },
     {
+      name: 'ember-1-12',
```

Comment:
I should note this is the same config I'm using elsewhere, i.e. `ember-prop-types`. I'm just copy/pasting it into other addons.